### DISCUSSION TITLE
Internal links with markdown

### DISCUSSION THREAD

**Discussion by KobyGam** [datapane/datapane#74] [General]
Hi.
I tried to configure some external links using markdown syntax (e.g: `dp.Text('[Link to different report](./another_report.html)')`) and it works great. However when trying to link to an existing title in my current report (e.g: `dp.Text('[Link to inner title](# MyTitle)')`) it doesn't work.
Is there any way to add internal links in the report?

Thanks.

**Comment by mands** [comment]
That's a great suggestion - thanks! This isn't currently supported, but it should be possible to support linking to other markdown titles/anchors in the same report - we'll add to our backlog and look into it.

**Reply by Bebio95** [reply]
Hi, in the same way, it could be interesting to have link/buttons to navigate between different dp.Page or to go back at the top of the page.
Thanks

**Reply by mands** [reply]
Yes, agreed! Adding Page next / prev buttons to the bottom of the report is something we're looking into adding right now

**Reply by Bebio95** [reply]
Hi, I thought I saw in an anoucement that this feature had been implemented, but I can't find where. Have I dreamed ?
